lib/frontend/src/lib.rs

```diff
@@ -1,12 +1,43 @@
 //! Cranelift IR builder library.
 //!
 //! Provides a straightforward way to create a Cranelift IR function and fill it with instructions
-//! translated from another language. Contains an SSA construction module that lets you translate
-//! your non-SSA variables into SSA Cranelift IR values via `use_var` and `def_var` calls.
+//! corresponding to your source program written in another language.
 //!
 //! To get started, create an [`FunctionBuilderContext`](struct.FunctionBuilderContext.html) and
 //! pass it as an argument to a [`FunctionBuilder`](struct.FunctionBuilder.html).
 //!
+//! # Source Language Variables and Cranelift IR values
+//!
+//! The most interesting feature of this API is that it provides a single way to deal with all your
+//! variable problems. Indeed, the [`FunctionBuilder`](struct.FunctionBuilder.html) struct has a
+//! type parameter `Variable` that should be instantiated with the type of your Source Language
+//! Variables (SLV). Then, through calling the functions
```

Comment:
It feels a bit awkward to introduce an acronym here, especially since as you mention below, this also applies to artifacts of translation, so "Source Level Variables" isn't quite the right term. What would you think about introducing these as "mutable" variables, and say that these can represent mutable variables either in the source or arising from the translation?